homeassistant/components/thermostat/ecobee.py

```diff
@@ -26,8 +47,21 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     _LOGGER.info(
         "Loading ecobee thermostat component with hold_temp set to %s",
         hold_temp)
-    add_devices(Thermostat(data, index, hold_temp)
-                for index in range(len(data.ecobee.thermostats)))
+
+    thermostats = []
+    for index in range(len(data.ecobee.thermostats)):
+        thermostats.append(Thermostat(data, index, hold_temp))
+    add_devices(thermostats)
+
+    def send_message_service(service):
+        """Send message to thermostats."""
+        message = service.data['message']
+        for thermostat in thermostats:
+            thermostat.send_message(message)
+
+    hass.services.register(DOMAIN, 'send_message', send_message_service,
```

Comment:
Can we scope this to the ecobee platform? So `send_message_ecobee`. Also do you think this might be better as a notify platform ?